src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -14,16 +14,33 @@
   </PropertyGroup>
 
   <Target Name="NpmRunBuild" BeforeTargets="BeforeBuild">
-      <Exec Command="npm install --prefix ./ @dynamods/splash-screen@latest" />
+      <Exec Command="npm pack @dynamods/splash-screen@latest" />
   </Target>
+
+    <Target Name="ExtractTGZFile" BeforeTargets="BeforeBuild">
```

Comment:
@filipeotero Curious if this is depending on any particular zipping software? If I remember correctly, Windows now comes with unzipping tool OOTB?

Reply:
Looks like it's coming with libarchive:
https://learn.microsoft.com/en-us/virtualization/community/team-blog/2017/20171219-tar-and-curl-come-to-windows